guacamole-common-js/src/main/webapp/modules/Keyboard.js

```diff
@@ -885,7 +885,12 @@ Guacamole.Keyboard = function Keyboard(element) {
         for (var i = 0; i < str.length; i++) {
 
             // Determine keysym of current character
-            var codepoint = str.codePointAt ? str.codePointAt(i) : str.charCodeAt(i);
+            var codepoint = str.codePointAt(i);
```

Comment:
I don't think we should remove the checks for `codePointAt` - it was only added in mid-2017.